packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanBarRow.tsx

```diff
@@ -150,7 +154,25 @@ export default class SpanBarRow extends React.PureComponent<SpanBarRowProps> {
               <span
                 className={`span-svc-name ${isParent && !isChildrenExpanded ? 'is-children-collapsed' : ''}`}
               >
-                {showErrorIcon && <IoAlert className="SpanBarRow--errorIcon" />}
+                {showErrorIcon &&
+                  (hasOwnError ? (
+                    <IoAlert className="SpanBarRow--errorIcon" />
+                  ) : (
+                    <button
+                      type="button"
+                      className="SpanBarRow--errorIcon SpanBarRow--errorIcon--bubbled"
+                      title="Click to navigate to descendant error span(s)"
```

Comment:
please include screenshots demonstrating the tooltip and the result of clicking on the icon